.github/workflows/cypress-workflow-bundle-snapshot-based-ci-groups.yml

```diff
@@ -72,7 +72,7 @@ jobs:
     uses: ./.github/workflows/release-e2e-workflow-template.yml
     with:
       test-name: "osd:ciGroup${{ matrix.ciGroup }}"
-      test-command: env CYPRESS_NO_COMMAND_LOG=1 CYPRESS_ML_COMMONS_DASHBOARDS_ENABLED=true CYPRESS_VISBUILDER_ENABLED=true CYPRESS_DATASOURCE_MANAGEMENT_ENABLED=true yarn cypress:run-with-security --browser chromium --spec '${{ matrix.specs }}'
```

Comment:
Is these `CYPRESS_NO_COMMAND_LOG` remove by intentionally? As I remeber we add this variable to unhang before.

Reply:
Yes, by reducing the defaultCommandTimeout to 10000, the test is able to run without hang. So I remove the command log so that the video can give more info for debugging.